WPMedia/arc-themes-components
# THEMES-1476: Dropdown input

## Ticket

- [THEMES-1476](https://arcpublishing.atlassian.net/browse/THEMES-1476)

## Description

Adds a 'select' option to the input component, expecting the same props as `FormInputField` from `@wpmedia/shared-styles`. This ensures that `input` can be a 1:1 replacement in our block code and in client custom code.

## Acceptance Criteria

- dropdown component is created to meet the attached designs

- unit tests are updated

## Test Steps

1. Checkout branch - `git checkout THEMES-1476`
2. Update dependencies - `npm i`
3. Since Storybook is down for now, create a small custom block in the feature pack by adding `/components/features/select-input/default.jsx` with the following code:
```
import { Input } from "@wpmedia/arc-themes-components";

const SelectInput = () => (
	<Input
		defaultValue="option1"
		name="test"
		label="label"
		options={[
			{ label: 'Option 1', value: 'option1' },
			{ label: 'Option 2', value: 'option2' },
		]}
		type="select"
	/>
);

export default SelectInput;
```
4. Edit a page and add the new test block.
5. The styling for the select box should match the Figma documentation.

## Author Checklist


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1476]: https://arcpublishing.atlassian.net/browse/THEMES-1476?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
